### PR TITLE
Update Godeps

### PR DESCRIPTION
**Pull request by dlackty**
Recently I started trying this project and found:
1. go-uuid packages was moved from Google Code to GitHub
2. honeybadger v0.0.1 also depended on that packages, so I updated it to newer v0.0.2.

### CODE REVIEW

**Comment by ejholmes** [comment]
Hey @dlackty, thanks for this! We've been trying to move our projects away from godeps and instead use the new Go 1.6 vendoring, which I think would solve the problem you probably ran into (I'm guessing `godep restore` fails?).

To manage deps, we've been using [govendor](https://github.com/kardianos/govendor) and migrating the Godeps folder with `govendor migrate`.

**Comment by dlackty** [comment]
Hey @ejholmes I've added a new commit to use govendor instead. Could you please have a look?

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-RUN go get github.com/tools/godep
+RUN go get -u github.com/kardianos/govendor
 
 ENV DOCKER_HOST unix:///var/run/docker.sock
 
 ADD . /go/src/github.com/remind101/conveyor
 WORKDIR /go/src/github.com/remind101/conveyor
-RUN godep go install ./cmd/conveyor
+RUN govendor install ./cmd/conveyor
```
Comment (ejholmes) [inline Dockerfile]: No need to install and use govendor. Just the standard `go install` will do.

**Comment by ejholmes** [comment]
I think CircleCI is still using Go 1.5.3, so we'll need to set `GO15VENDOREXPERIMENT=1` in the environment.

**Comment by dlackty** [comment]
Both updated, and CI became green as well.

**Comment by dlackty** [comment]
this should also close #61

**Comment by bronislav** [comment]
@ejholmes Any updates on this?

**Comment by ejholmes** [comment]
I pulled this in with https://github.com/remind101/conveyor/commit/ad2254bbcfb644f40a6923b1cd4fcbd04235b48a, so master is build-able without godep.

**Pull request closed without merging**